### ISSUE TITLE
No suggestions

### ISSUE THREAD

**Issue by coret** [omeka-s-modules/NdeTermennetwerk#3]
Whatever source I use, the results are always empty:
```
{"suggestions":[]}
```

I've added some logging [src/Suggester/NdeTermsSuggest.php](https://github.com/omeka-s-modules/NdeTermennetwerk/blob/main/src/Suggester/NdeTermsSuggest.php) in the pareResult function and see that this function gets as $result:
```
Array
(
    [data] =>
    [errors] => Array
        (
            [0] => Array
                (
                    [message] => Cannot read properties of undefined (reading 'operationName')
                )

        )

)
```

**Comment by coret** [comment]
I found a solution:
https://github.com/omeka-s-modules/NdeTermennetwerk/blob/9db5caae70e39c909ab2d746d21949f2b6fd9cf4/src/Suggester/NdeTermsSuggest.php#L106

if I replace this line with `->setRawBody(json_encode(['query' => $query]))`, I do get results again from the GraphQL API.

@ddeboer has the GraphQL API of the NoT become stricter? Did the API ever support the GET method?

**Comment by coret** [comment]
Fixed in release 1.0.1
